code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -439,20 +439,23 @@ im not even gonna bother with these for the following symptoms. typed em out, co
 		if(4, 5)
 			if(burnheal)
 				M.heal_overall_damage(0, 1.5) //no required_status checks here, this does all bodyparts equally
-			if(prob(5) && (M.bodytemperature < BODYTEMP_HEAT_DAMAGE_LIMIT || M.bodytemperature > BODYTEMP_COLD_DAMAGE_LIMIT))
+			if(!cooldowntimer && (M.bodytemperature < BODYTEMP_HEAT_DAMAGE_LIMIT || M.bodytemperature > BODYTEMP_COLD_DAMAGE_LIMIT) && (!location_return || location_return.z != M.loc.z))
 				location_return = get_turf(M)	//sets up return point
-				if(prob(50))
-					to_chat(M, "<span class='userwarning'>The lukewarm temperature makes you feel strange!</span>")
-			if(cooldowntimer == 0 && ((M.bodytemperature > BODYTEMP_HEAT_DAMAGE_LIMIT + telethreshold  && !HAS_TRAIT(M, TRAIT_RESISTHEAT)) || (M.bodytemperature < BODYTEMP_COLD_DAMAGE_LIMIT - telethreshold  && !HAS_TRAIT(M, TRAIT_RESISTCOLD)) || (burnheal && M.getFireLoss() > 60 + telethreshold)))
+				to_chat(M, "<span class='warning'>The lukewarm temperature makes you feel strange!</span>")
+				cooldowntimer = (300 + rand(1, 300))
```

Comment:
please replace this with cooldown defines

Reply:
can't get this to work with cooldown defines. tried my best